index.js

```diff
@@ -1,13 +1,20 @@
-const { Command } = require('commander');
-const lib = require('./lib/dmap/dmap.js')
-const rpc = require('./rpc.js')
+import { Command } from 'commander'
+import  lib  from './lib/dmap/dmap.js'
+import { rpc }  from './rpc.js'
+import { jams } from './lib/jams/jams.js'
```

Comment:
This should be importing from node modules instead. I think there's a bug in jams which prevented imports working, I added a jams PR now to fix it

Reply:
addressed